Branch.podspec

```diff
@@ -15,26 +15,31 @@ Use the Branch SDK (branch.io) to create and power the links that point back to
   s.homepage         = "https://branch.io"
   s.license          = 'Proprietary'
   s.author           = { "Branch" => "support@branch.io" }
-  s.source           = { :git => "https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK.git", :tag => s.version.to_s }
+  s.source           = { git: "https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK.git", tag: s.version.to_s }
   s.platform         = :ios, '7.0'
   s.requires_arc     = true
 
+  source_files =
```

Comment:
Nevermind. Each subspec requires at minimum source_files or a number of other fields not present here. These have to stay, so using the local variable is probably the easiest thing to do. The private_header_files can probably belong to the root spec, however.